x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/CertificateTool.java

```diff
@@ -274,6 +278,23 @@ final void acceptInputFile() {
             inputFileSpec = parser.accepts("in", "file containing details of the instances in yaml format").withRequiredArg();
         }
 
+        final void acceptCertificateAuthorityKeyUsage() {
+            OptionSpecBuilder builder = parser.accepts(
+                "ca-keyusage",
```

Comment:
If we only support this on `certutil ca` then we can just rename it to `--keyusage` (drop the "ca-" prefix)